packages/components/src/components/hds/badge/index.ts

```diff
@@ -4,39 +4,44 @@
  */
 
 import Component from '@glimmer/component';
-import { assert } from '@ember/debug';
-
-export const DEFAULT_SIZE = 'medium';
-export const DEFAULT_TYPE = 'filled';
-export const DEFAULT_COLOR = 'neutral';
-export const SIZES = ['small', 'medium', 'large'];
-export const TYPES = ['filled', 'inverted', 'outlined'];
-export const COLORS = [
-  'neutral',
-  'neutral-dark-mode',
-  'highlight',
-  'success',
-  'warning',
-  'critical',
-];
-
-export default class HdsBadgeIndexComponent extends Component {
+import {assert} from '@ember/debug';
+import {type Color, ColorValues, type Size, SizeValues, type Type, TypeValues} from "./types.ts";
```

Comment:
In the same spirit with Cristiano's comment above, we'd prefer these written as below (cascading changes needed) as [we try to align these across components](https://docs.google.com/document/d/17T9v7pYUNrm2TtcZE2le_fbX30Jop-iG3um7b0trRlg/edit#bookmark=id.kw9kdqlfg87r).
```suggestion
import {HdsBadgeColorValues, HdsBadgeSizeValues, HdsBadgeTypeValues} from "./types.ts";
import type { HdsBadgeColors, HdsBadgeSizes, HdsBadgeTypes } from "./types.ts";
```